src/main/java/com/minerarcana/floralchemy/WorldEventListener.java

```diff
@@ -0,0 +1,75 @@
+package com.minerarcana.floralchemy;
+
+import net.minecraft.block.state.IBlockState;
+import net.minecraft.entity.Entity;
+import net.minecraft.entity.player.EntityPlayer;
+import net.minecraft.init.Blocks;
+import net.minecraft.util.SoundCategory;
+import net.minecraft.util.SoundEvent;
+import net.minecraft.util.math.BlockPos;
+import net.minecraft.world.*;
+
+public class WorldEventListener implements IWorldEventListener {
+
+    private WorldServer world;
+    
+    public WorldEventListener(WorldServer world) {
+        this.world = world;
+    }
+
+    @Override
+    public void notifyBlockUpdate(World worldIn, BlockPos pos, IBlockState oldState, IBlockState newState, int flags) {
+    }
+
+    @Override
+    public void notifyLightSet(BlockPos pos) {
+    }
+
+    @Override
+    public void markBlockRangeForRenderUpdate(int x1, int y1, int z1, int x2, int y2, int z2) {
+    }
+
+    @Override
+    public void playSoundToAllNearExcept(EntityPlayer player, SoundEvent soundIn, SoundCategory category, double x,
+            double y, double z, float volume, float pitch) {
+    }
+
+    @Override
+    public void playRecord(SoundEvent soundIn, BlockPos pos) {
+    }
+
+    @Override
+    public void spawnParticle(int particleID, boolean ignoreRange, double xCoord, double yCoord, double zCoord,
+            double xSpeed, double ySpeed, double zSpeed, int... parameters) {
+    }
+
+    @Override
+    public void spawnParticle(int id, boolean ignoreRange, boolean minimiseParticleLevel, double x, double y, double z,
+            double xSpeed, double ySpeed, double zSpeed, int... parameters) {
+    }
+
+    @Override
+    public void onEntityAdded(Entity entityIn) {
+    }
+
+    @Override
+    public void onEntityRemoved(Entity entityIn) {
+    }
+
+    @Override
+    public void broadcastSound(int soundID, BlockPos pos, int data) {
+    }
+
+    //Bahahaha
+    @Override
+    public void playEvent(EntityPlayer player, int type, BlockPos blockPosIn, int data) {
+        if(type == 1031 && world.getBlockState(blockPosIn.down()).getBlock() == Blocks.CAULDRON) {
```

Comment:
Isn't Registry Replacement better than trying to catch and replace everything? Also do you have a config for this?

Reply:
You mean replacing the anvil block? I could do that, but what's the problem with the current way. It's not too bad efficiency wise.

What would you suggest the alternative is when this is config-ed off? Or leave the user to setup the alternative?